.circleci/config.yml

```diff
@@ -85,6 +87,39 @@ jobs:
           command: go test -a -timeout=15m .
           no_output_timeout: 15m
 
+  test-build-tamago-arm:
+    docker:
+      - image: uroottest/test-image-tamago:v4.4.0
+    working_directory: /home/circleci/go/src/github.com/u-root/u-root
+    environment:
+      - UROOT_SOURCE: /home/circleci/go/src/github.com/u-root/u-root
+      - CGO_ENABLED: 0
+    steps:
+      - checkout
+      - run:
+          name: Test u-root tamago build
+          command: GOOS=tamago GOARCH=arm GOARM=7 go build ./pkg/dt/...
+
+  test-build-tamago-riscv64:
```

Comment:
armory-boot is not supported under riscv64, so this test doesn't make much sense to me.

Reply:
move it to arm I guess? This is mainly a did we break the build but your'r'e right :-)